kml/edit-kml-ground-overlay/src/main/java/com/esri/samples/edit_kml_ground_overlay/EditKMLGroundOverlaySample.java

```diff
@@ -0,0 +1,162 @@
+
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      fxScene.getStylesheets().add(getClass().getResource("/style.css").toExternalForm());
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Edit KML Ground Overlay Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene and add it to the scene view
+      ArcGISScene scene = new ArcGISScene(Basemap.createImagery());
+      sceneView = new SceneView();
+      sceneView.setArcGISScene(scene);
+
+      // create a geometry for the ground overlay
+      Envelope overlayGeometry = new Envelope(-123.066227926904, 44.04736963555683, -123.0796942287304, 44.03878298600624, SpatialReferences.getWgs84());
+
+      // create a KML icon for the overlay image
+      String overlayImageURI = new File("./samples-data/images/1944.jpg").getAbsolutePath();
+      KmlIcon overlayImage = new KmlIcon(overlayImageURI);
+
+      // create the KML ground overlay
+      KmlGroundOverlay kmlGroundOverlay = new KmlGroundOverlay(overlayGeometry, overlayImage);
+
+      // set the rotation of the ground overlay
+      kmlGroundOverlay.setRotation(-3.046024799346924);
+
+      // create a KML dataset with the ground overlay as the root node
+      KmlDataset kmlDataset = new KmlDataset(kmlGroundOverlay);
+
+      // create a KML layer using the KML dataset
+      KmlLayer kmlLayer = new KmlLayer(kmlDataset);
+
+      // add the KML layer to the scene view
+      sceneView.getArcGISScene().getOperationalLayers().add(kmlLayer);
+
+      // set the viewpoint to the ground overlay
+      sceneView.setViewpoint(new Viewpoint(kmlGroundOverlay.getExtent(), new Camera(kmlGroundOverlay.getGeometry().getExtent().getCenter(), 1250, 45, 60, 0)));
+
+      // create a slider for adjusting the overlay opacity
+      Slider slider = new Slider(0, 1, 1);
+      slider.setMaxWidth(300);
+
+      // add an event handler to the slider to apply opacity to the KML ground overlay based on the slider value
+      slider.setOnMouseReleased(event -> updateGroundOverlayOpacity(kmlGroundOverlay, slider.getValue()));
+      slider.setOnMouseDragged(event -> updateGroundOverlayOpacity(kmlGroundOverlay, slider.getValue()));
+
+      // create a controls box
+      VBox controlsVBox = new VBox();
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.5)"), CornerRadii.EMPTY,
+              Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setSpacing(10);
+      controlsVBox.setMaxSize(250, 20);
+      controlsVBox.getStyleClass().add("panel-region");
+
+      // create a label for the slider
+      Label label = new Label("Overlay Opacity: ");
```

Comment:
`Label label = new Label(" Overlay Opacity");`

Adding an extra space before "Overlay" will line up the text with the slider so that it's aligned, and we also don't need the colon.

Reply:
I don't think adding a space is going to be a very robust way to line up controls. Probably you can set some kind of alignment property on the `VBox` - https://openjfx.io/javadoc/11/javafx.graphics/javafx/scene/layout/VBox.html#setAlignment(javafx.geometry.Pos) maybe.

Do you test the samples on a right to left layout?

Reply:
that's true, I keep forgetting we have other platforms like Mac that can display text alignment differently. So a padding option or the above could be a better approach. If that still proves fiddly, perhaps centre align the label text above the slider to avoid the issue.

Reply:
and no, we don't usually test on a right to left layout, at least not that I'm aware of: not sure if this is something @hardikc07 tests for.

Reply:
Using RTL is a good way to see if the controls are laid out properly e.g. the extra space would possibly end up on the wrong side.

Reply:
The previous version had lefthand-alignment.
After playing around with it a little I've opted to just center both elements in the VBbox, so it's LH/RH agnostic.